mkdoxy/doxyrun.py

```
@@ -155,17 +156,27 @@ def str2dox_dict(self, dox_str: str) -> dict:
         @return: (dict) Dictionary.
         """
         dox_dict = {}
+        dox_str = re.sub(r'\\\s*\n\s*', '', dox_str)
+        pattern = r'^\s*([^=\s]+)\s*(=|\+=)\s*(.*)$'
+
         try:
             for line in dox_str.split("\n"):
-                if line.strip() == "":
+                if line.strip() == "" or line.startswith("#"):
+                    continue
+                match = re.match(pattern, line)
+                if not match:
                     continue
```

Maybe better to not silently swallow lines if we cannot match? Guess this should at least print the line, maybe throw even.

---

Sure, I can add that.  Does @JakubAndrysek have a preference?

---

I agree with @julianeisel. Could you please update that? Thanks

---

Done!